Readme.md

```diff
@@ -107,6 +107,7 @@ The available options are:
     default not set.
   * `phantomPath`: If PhantomJS is not installed in your path,
     you can use this option to specify the executable's location.
+  * `phantomOptions`: additional PhantomJS options.
```

Comment:
There should probably be some sort of disclaimer that this can override horseman's choices of Phantom options.